install/kubernetes/values.yaml

```diff
@@ -160,25 +160,34 @@ export:
   # "stdout". "" to disable.
   mode: "stdout"
   resources: {}
-  extraArgs: {}
-  extraEnv: []
-  # extraEnv:
-  #   - name: foo
-  #     value: bar
-  extraVolumeMounts: []
   securityContext: {}
 
-  # Override the command. For advanced users only.
-  commandOverride: [ ]
-
-  # Override the arguments. For advanced users only.
-  argsOverride: [ ]
-
   # filenames defines list of files for fluentd to tail and export.
   filenames:
   - tetragon.log
 
+  # stdout specific exporter settings
   stdout:
+    extraEnv: []
+    # extraEnv:
+    #   - name: foo
+    #     value: bar
+
+    # should the manifest command be overriden?
+    enabledCommand: true
+
+    # should the manifest args be overridden?
+    enabledArgs: true
```

Comment:
Shouldn't these be set to false by default?

Reply:
i.e. we want the default behaviour to be "use command and args from the export-stdout container".

Reply:
Yes, happy to reverse the defaults.